src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -589,7 +593,83 @@ <h2 class="modal-title">
           </div>
         </div>
 
-        <div class="row" ng-show="vm.field.fieldType === 'email'"><br /></div>
+        <div class="row" ng-show="vm.field.fieldType === 'email'">
+          <br />
+          <div class="toggle-option">
+            <div
+              ng-class="{'toggle-option--disabled' : !vm.field.isVerifiable}"
+              class="col-xs-8 label-custom label-medium"
+            >
+              Restrict email domains<i
+                class="glyphicon glyphicon-question-sign"
+                uib-tooltip="e.g. @mom.gov.sg, @moe.gov.sg OTP verification needs to be activated first."
+                tooltip-trigger="'click mouseenter'"
+              ></i>
+            </div>
+            <div class="col-xs-4 field-input">
+              <label
+                class="toggle-selector pull-right"
+                ng-class="{'toggle-selector-on': vm.field.isVerifiable && vm.field.hasAllowedEmailDomains, 'toggle-selector--disabled': !vm.field.isVerifiable}"
+                onclick=""
```

Comment:
Is this empty onclick needed? Remove if not needed, comment on purpose if needed

Reply:
Based on [this](https://github.com/ghinda/css-toggle-switch#browser-support), yes. Plus, existing code above also has this empty `onclick`. But I vaguely recall that our browser support does not extend to those? Let me know what's the best action to take

Reply:
Got it. Should keep to the status quo then. Mind adding a comment with that link at the onclick so other devs won't touch it?

Reply:
Ok